po/common/ru.po

```diff
@@ -815,7 +828,7 @@ msgstr "Общепринятое название"
 
 msgctxt "generic_name_example"
 msgid "Chocolate bar with milk and hazelnuts"
-msgstr "Шоколадная плитка с молоком и фундуком"
+msgstr "Молоко питьевое ультрапастеризованное, массовая доля жира 2,5%; Зубная паста; Корм сухой полнорационный для кошек старше 7 лет, с курицей"
```

Comment:
That looks more like a list of ingredients

Reply:
- Those are specific examples for Open Pet Food Facts. We show dog food brands instead of human food brands.
- @aleksejrs initially concatenated exemples from OFF, OBF and OPFF. He needs to update the translation within Crowdin

Reply:
corrected in Crowdin